minepi/player.py

```diff
@@ -260,7 +260,10 @@ async def initialize(self):
                         self._raw_cape = Image.open(BytesIO(await resp.read()))
                 else:
                     self._raw_cape = None
-
+                    
+        if self._raw_cape != None:
```

Comment:
```suggestion
        if self._raw_cape is not None and self._raw_cape.mode != "RGBA":
```

Reply:
As mentioned in discord, i would prefer to have it check the mode before converting